slave-base/contrib/bin/run-jnlp-client

```diff
@@ -96,11 +96,12 @@ if [[ $# -gt 1 ]]; then
     fi
 
     JAVA_MAX_HEAP_PARAM="-Xmx${CONTAINER_HEAP_MAX}m"
-    if [ -z $CONTAINER_INITIAL_PERCENT ]; then
-      CONTAINER_INITIAL_PERCENT=0.07
+    if [[ "$CONTAINER_INITIAL_PERCENT" ]]; then
+      CONTAINER_INITIAL_HEAP=$(echo "${CONTAINER_HEAP_MAX} ${CONTAINER_INITIAL_PERCENT}" | awk '{ printf "%d", $1 * $2 }')
+      if [[ -z "$JAVA_INITIAL_HEAP_PARAM" ]]; then
+        JAVA_INITIAL_HEAP_PARAM="-Xms${CONTAINER_INITIAL_HEAP}m"
+      fi
```

Comment:
so if I caught the gist of the recent exchanges with Severin correctly (not tagging him on purpose to save him some additional chatter) the gist was that the JVM will do a better job at picking the default when used in conjunction with the honor cgroup flags, right?

if so, a comment explaining all that would be good.

Reply:
The JVM default is Xms=1/64 of Xmx, the 64 is configurable using a JVM flag.  Severin made a change to stop us setting Xms==Xmx and set it to 7% instead.  In fact I think he could have gone further, I think there's no harm in using the JVM default here.

Reply:
thanks for that ...now, maybe it wasn't clear ... I meant a comment in the code